samples/tls/logging/README.md

```diff
@@ -37,9 +24,26 @@ my_repository_path/logging/bin/deploy_logging_open.sh
 ```
 ## Notes On Customization Values
 
-Set `MON_TLS_ENABLE=true` in the `$USER_DIR/logging/user.env` file. This variable modifies the deployment of Open Distro for Elasticsearch to be TLS-enabled.
+### Ingress
+
+* If you are using ingress, set the environment variable `TLS_ENABLED=true` in the `user.env` file in order to use TLS for connections between the user and Kibana.  
+
+* Manually populate these secrets in the `logging` namespace (or `LOG_NS` value) **before**
+you run the deployment script:
+
+  * kubernetes.io/tls secret - `kibana-ingress-tls-secret`
+  * kubernetes.io/tls secret - `elasticsearch-ingress-tls-secret`
+
+Generating these certificates is outside the scope of this example. However, you
+can use the process documented in ["Configure NGINX Ingress TLS for SAS Applications"](https://go.documentation.sas.com/?cdcId=sasadmincdc&cdcVersion=default&docsetId=calencryptmotion&docsetTarget=n1xdqv1sezyrahn17erzcunxwix9.htm&locale=en#n0oo2yu8440vmzn19g6xhx4kfbrq) in SAS Viya Administration documentation and specify the `logging` namespace.
+
+If any of the required certificates do not exist, the deployment process attempts to use [cert-manager](https://cert-manager.io/) to generate the missing certificates.
+
+* If you are using an ingress controller other than NGINX, modify the annotation 
+`nginx.ingress.kubernetes.io/backend-protocol: HTTPS` in the `user-values-elasticsearch-open.yaml` file. Refer to the documentation for your ingress controller. 
+
+### Nodeports
+
+If you are using nodeports,  set the environment variable `TLS_ENABLED=true` in the `user.env` file in order to use TLS for connections between the user and Kibana. 
 
```

Comment:
The Nodeports section can probably be removed if/when the sentence about setting TLS_ENABLED=true" if moved out of the Ingress section (since, as commented above, the setting has effect even if ingress is not enabled).  Hmmm, we need to look into whether the es_nodeport_enable.sh script needs to be tweaked to do something different if TLS_ENABLED is true (i.e. can we force HTTPS for inbound calls to Elasticsearch?).

Reply:
After giving this a bit more thought, how about doing the following:

- Add a new section (under Customization) called TLS, and explain how to set the TLS_ENABLE env var to "true" to enable TLS (in the one spot it isn't being used).
- The Ingress section should only talk about the need for those annotations and the need to populate the secrets with the actual TLS certs.
- Remove the Nodeport section.

I think that's cleaner and would make more sense to the reader.

Reply:
That does make the organization cleaner. Changed as noted.